packages/at_secondary_server/test/stats_verb_test.dart

```diff
@@ -124,7 +124,7 @@ void main() {
   group('A group of notificationStats verb tests', () {
     SecondaryKeyStoreManager? keyStoreManager;
     setUp(() async => keyStoreManager = await setUpFunc(
-        '${Directory.current.path}/test/hive',
+        '${Directory.current.path}/unit_test_storage',
```

Comment:
why the change? (just curious)

Reply:
At the end of the group added a tearDown method from test_utils.dart - `tearDown(() async => await verbTestsTearDown());` which clears from "unit_test_storage". Hence marked the storage directory to "unit_test_storage" to ensure the test data is cleared at the end of the test.